app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/components/ClustersStepForm/ClustersStepForm.js

```diff
@@ -151,7 +164,7 @@ class ClustersStepForm extends React.Component {
                   text={
                     item.v_parent_datacenter
                       ? `${item.ext_management_system.name} \\ ${item.v_parent_datacenter} \\ ${item.name}`
-                      : `${item.name}`
+                      : `${targetProvider} \\ ${item.name}`
```

Comment:
If we use `item.ext_management_system.name` instead of `targetProvider`, we'll get `OpenStack` instead of `openstack`.

Also, in the strange case when a rhevm cluster has no `v_parent_datacenter`, this happens
<img width="333" alt="manageiq__migration" src="https://user-images.githubusercontent.com/15141412/43545988-6b5fd986-95a5-11e8-854c-a6364c12a256.png">

Reply:
What's the this?

Reply:
oh, sorry, I missed my GH notification.  I was just pointing out that the top provider prefix is `RHV` while the bottom one is `rhevm`.   But now it's consistently `RHV`

Reply:
Yep!! it took a few seconds for that to click 😬 😴  (must be that time)